templates/nas-grp-base.html.twig

```diff
@@ -159,8 +159,8 @@
         });
     </script>
 
-    <script>
-        window.PELAGOS_TEMPLATE_PROPS = {"hideSearchTextBlock": true, "fundingCycleFacet": true};
+    <script type="text/javascript">
+        window.PELAGOS_TEMPLATE_PROPS = {"BaseTemplateName": "GRP", "hideSearchTextBlock": true, "fundingCycleFacet": true};
```

Comment:
Because `fundingCycleFacet` kinda does the same thing, you could key of that. Maybe even rename it to something common.